### PR TITLE
#5756 [Manager] Unable to terminate services after running cypress te…

### PR DESCRIPTION
**Pull request by konekoya**
…sts in headless mode

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/scripts/runTests.js [hunk 1]
```diff
@@ -56,7 +58,21 @@ const run = async (ci, apiRoot, serverPort = 5050, clientPort = 3000) => {
   console.log(chalk.blue('Starting ohara manager server'));
   server = execa(
     'forever',
-    ['start', 'start.js', '--configurator', apiRoot, '--port', serverPort],
+    [
+      'start',
+      '--uid',
+      SERVER_UID,
```
Comment (konekoya) [inline ohara-manager/scripts/runTests.js:64]: Assign a `UID` for the process and so we can later use this `UID` to kill the process. `server.kill()` is not able to kill processes started by forever

File: ohara-manager/scripts/runTests.js [hunk 2]
```diff
@@ -56,7 +58,21 @@ const run = async (ci, apiRoot, serverPort = 5050, clientPort = 3000) => {
...
+      '--append',
+      '--minUptime',
+      1000,
+      '--spinSleepTime',
+      1000,
```
Comment (konekoya) [inline ohara-manager/scripts/runTests.js:69]: Get rid of the warnings as well

File: ohara-manager/scripts/runTests.js [hunk 3]
```diff
@@ -109,21 +132,11 @@ const run = async (ci, apiRoot, serverPort = 5050, clientPort = 3000) => {
...
-    ],
-    {
-      stdio: 'inherit',
-    },
-  );
-
   // We need these jars for test
   try {
-    await copyJars;
+    execa.sync('yarn', ['copy:jars', '--ci'], {
```
Comment (konekoya) [inline ohara-manager/scripts/runTests.js:137]: copyJars can be run in sync and doesn't need to be cleaned

File: ohara-manager/client/yarn.lock [hunk 4]
```diff
@@ -13188,11 +13188,6 @@ utils-merge@1.0.1:
   resolved "https://registry.yarnpkg.com/utils-merge/-/utils-merge-1.0.1.tgz#9f95710f50a267947b2ccc124741c1028427e713"
   integrity sha1-n5VxD1CiZ5R7LMwSR0HBAoQn5xM=
 
-uuid@8.3.0:
```
Comment (konekoya) [inline ohara-manager/client/yarn.lock:13191]: It's now used by manager too, so removing from client deps and move to manager

**Comment by konekoya** [comment]
retry manager-it

**Review by eechih — APPROVED** [review]

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**